xp tracker makes client freeze when hopping to/from deadman world

```
2019-08-15 18:45:11 [Client] ERROR net.runelite.client.RuneLite - Uncaught exception:
java.lang.AssertionError: null
	at net.runelite.client.plugins.xptracker.XpTrackerPlugin.onGameStateChanged(XpTrackerPlugin.java:194)
	at net.runelite.client.eventbus.EventBus$Subscriber.invoke(EventBus.java:72)
	at net.runelite.client.eventbus.EventBus.post(EventBus.java:218)
	at net.runelite.client.callback.Hooks.post(Hooks.java:161)
	at client.kv(client.java:5002)
	at cv.ei(cv.java:1105)
	at gb.gi(gb.java:5372)
	at client.ad(client.java:920)
	at bz.bo(bz.java:364)
	at bz.run(bz.java:343)
	at java.base/java.lang.Thread.run(Thread.java:834)

```

--- a/runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerPlugin.java
+++ b/runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerPlugin.java
@@ -195,11 +195,11 @@ public class XpTrackerPlugin extends Plugin
 				fetchXp = true;
 				lastWorldType = type;
 				resetState();
-				// Must be set from hitting the LOGGING_IN case below
+				// Must be set from hitting the LOGGING_IN or HOPPING case below
 				assert initializeTracker;
 			}
 		}
-		else if (state == GameState.LOGGING_IN)
+		else if (state == GameState.LOGGING_IN || state == GameState.HOPPING)
 		{
 			initializeTracker = true;
 		}

